terraform/modules/data-share-service/cloudwatch.tf

```diff
@@ -201,7 +201,7 @@ module "request_alarm_dashboard" {
   ]
 }
 
-module "queue_alarm_dashboard" {
+module "internal_alarm_dashboard" {
```

Comment:
did you mean to add the alarm here?

Reply:
Lol I ammended my commit and then to add it and then forgot to force push 🙃